translations/nn.json

```diff
@@ -70,13 +70,14 @@
       },
       "antallDager": {
         "dager": "dagar",
-        "label": "Skriv inn antall dager du skal ha ferie",
+        "label": "Skriv inn antall arbeidsdagar du skal ta ferie",
+        "description": "Lørdagar, søndagar og hellidagdagar skal ikkje føres inn som ferie.",
         "validation": {
-          "required": "Du må skrive inn antall dager du skal ha ferie."
+          "required": "Du må skrive inn antall arbeidsdagar du skal ta ferie."
         }
       },
       "alert": {
-        "text": "Det er viktig at du gir oss beskjed hvis du bestemmer deg for å ta ferie, fordi det kan påvirke startdatoen din. Dette påvirker spesielt deg som får sykepenger."
+        "text": "Det er viktig at du gir oss beskjed hvis du bestemmer deg for å ta ferie, fordi det kan påvirke startdatoen din."
```

Comment:
Det er viktig at du gir oss beskjed **viss** du bestemmer deg for å ta ferie, fordi det kan **påverka** startdatoen din.